### DISCUSSION TITLE
Hello, how do I set the port after webrtc-streamer is started? How do I set webrtc-streamer to start automatically under Windows?

### DISCUSSION THREAD

**Discussion by PHJasd** [mpromonet/webrtc-streamer#570] [General]
Hello, how do I set the port after webrtc-streamer is started? How do I set webrtc-streamer to start automatically under Windows?

**Comment by mpromonet** [comment]
Hi,

The listening port is decided at startup, you cannot change while it is running. 
To start automatically you just need to run the executable or use the restart-policy of docker.

Best Regards,
Michel.
